tools/lang/Geogebra.ggb

```diff
@@ -0,0 +1,1260 @@
+// https://github.com/geogebra/geogebra
+// https://geogebra.github.io/docs/manual/en/
+
+
+//! operators		===========================================================
++ - * / ^ ! ( ) [ ] { } = : | ! & - < > \  % ? . , '' $ " "
```

Comment:
I think double quote `"` means C/C++ like string/text instead of operator.

Reply:
Fixed this mistake.